cookieplone/utils/commands/__init__.py

```diff
@@ -18,6 +18,8 @@ def _get_command_version(cmd: str) -> str:
         )
     except FileNotFoundError:
         raw_version = ""
+    except PermissionError:
+        raw_version = ""
```

Comment:
This is fine, but you could also change line 19 to `except (FileNotFoundError, PermissionError):`